NitroxClient/Communication/Abstract/IClient.cs

```diff
@@ -8,7 +8,7 @@ namespace NitroxClient.Communication.Abstract
     public interface IClient
     {
         bool IsConnected { get; }
-        void Start(string ipAddress);
+        void Start(string ipAddress, int ServerPort);
```

Comment:
Method parameters should be camcelCase. `serverPort`

Reply:
Should I change it to serverPort

Reply:
Yes, that's the naming convention for method parameters.